.env

```diff
@@ -0,0 +1 @@
+VITE_API_KEY=731a6fa24765316e8e44e6e18157a6bd
```

Comment:
generally you don't check .env into your repo cause it exposes things that shouldn't be public like keys. you would want to put this in your .gitignore